Result for Max Life Span.Rmd

```diff
@@ -0,0 +1,8 @@
+---
+title: "Result for Max Life Span"
+output: html_document
+---
+
+## Maximum Life Span
+
+Figure 3, generated by the linear mixed model effect shows us that there is a negative correlation between number of offspring produced per year and the life span of the mother. Using orders as the random effect, the linear mixed effects model was a better fit for the data with a lower AIC rank of 714.4 in comparison to the generalized linear model which had a AIC rank of 837.0 (Table 1). Though this negative correlation is seen for all 7 mammalian orders, the correlation for some orders were stronger since their slopes were more negative, such *Carnivora* (-1.2317), *Primates* (-1.01260), and *Artiodactyla* (-0.6997) (Table 1). 
```

Comment:
Instead of saying "Figure 3 shows us...", state what your study revealed in the data and reference figure 3.

It could be interesting to comment on the magnitude of the slopes in the discussion or in methods. What is a significant difference between slopes? What magnitude is notable?